wlib/AttrListDlg.cpp

```diff
@@ -374,10 +374,16 @@ IModule * DoConfirmImportDlg(HWND hwndParent, const boost::filesystem::path & pa
 		std::_tstring attributeExt;
 		std::string attributeComment;
 		std::string attributeEcl;
+		bool firstline = true;
 
 		std::string line;
 		while (!safeGetline(ifs, line).eof())
 		{
+			if (firstline)
+			{
+				line = line.substr(3);
+				firstline = false;
+			}
```

Comment:
I think the correct solution would be to use CUnicodeFile to read the file - then it will take care of the BOM automatically?  The issue here, is that not all files have a BOM...

Reply:
@Gordon you are correct! I thought the CUnicodeFile had that at the front of any opened file but not true. I will correct this.